packages/core/src/dialects/abstract/query-interface.types.ts

```diff
@@ -42,3 +43,18 @@ export interface DescribeTableOptions extends QueryRawOptions {
    */
   schemaDelimiter?: string;
 }
+
+/** Options accepted by {@link AbstractQueryInterface#truncateTable} */
+export interface TruncateTableOptions extends QueryRawOptions, TruncateTableQueryOptions { }
+
+/** Options accepted by {@link AbstractQueryInterface#bulkDelete} and {@link AbstractQueryInterface#delete} */
+export interface DeleteOptions<TAttributes = any> extends QueryRawOptions, DeleteQueryOptions<TAttributes> {
```

Comment:
We have some unifying to do regarding the name of the types of the options accepted by query interface methods

They can easily collide with Model method options, and they're both exported by the package. I think it's best if we keep a prefix for query interface method option types. That's why some of them start with `Qi` (but not all because it's easy to let some slip through during a review) and query generator method option types include the word Query

Here it would be too easy to mistake DeleteOptions (QI) with DestroyOptions (model), TruncateTableOptions (QI) with TruncateOptions (model)

Reply:
I've updated the query options to `BulkDeleteOptions`